### ISSUE TITLE
Create new privacy policy page

### ISSUE THREAD

**Issue by R-Tomas-Gonzalez** [hackforla/website#2555]
### Dependency
- [x]  ~#2690~

### Overview
Create a new privacy policy page to properly inform users about how the website collects, uses, stores, and protects data from site visitors.

### Details
We're creating a new privacy policy page. The content can be found in the google drive and the design can be found in the figma design under the resources/instructions section.

### Action Items

- [x] Create a credit for privacy policy page header image based on the figma design
  - [x] Add a privacy-policy.yml file to the [/_data/internal/credits](https://github.com/hackforla/website/tree/gh-pages/_data/internal/credits) directory
  - [x] Follow the structure of other yml credit files and figma to create same structure
- [x] Create a new privacy-policy.html under the [/pages](https://github.com/hackforla/website/tree/gh-pages/pages) directory
- [x] Create a corresponding _privacy-policy.scss file under [/_sass/components](https://github.com/hackforla/website/tree/gh-pages/_sass/components) directory
- [x] Add the url to this [label description](https://github.com/hackforla/website/labels?q=Privacy+Policy)
- [x] Make sure the page looks fine for smaller viewport screens as well, such as tablet, mobile (responsiveness)

### Resources/Instructions
[HFLA Figma - Policy Page](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17845%3A137765)
[Content(Google Doc)](https://docs.google.com/document/d/19x1KsMTU-QBdopiV08HSBQ0i4g1ChrcPZ8ZXmUK50cY/edit#heading=h.1kvk4qkoxhzw)

Issue Creator - [Tomas Gonzalez - Slack DM](https://hackforla.slack.com/team/U02DF1PPQUA)
Lead Designer - [Isaac Cruz - Slack DM](https://hackforla.slack.com/team/U0255408P8Q)
Feature Page Designer - [Anh Hoang - Slack DM](https://hackforla.slack.com/team/U025DUAKAPK)

#### List of scss classes used in the Privacy Policy page (that are not in _privacy-policy.scss) and where they came from
- _sass/elements/_containers.scss
  - .header-container
    - .header-text
      - .sub-p
    - .header-hero-image 
  - .flex-container
- _sass/elements/_cards.scss
  - .page-card
  - .card-primary
  - .page-card-lg
- _sass/elements/_typography.scss
  - .title1
  - .title2
  - .paragraph2
- _sass/layouts/_main.scss
  - .content-section

#### Additional notes
- The mobile and desktop Figma designs are missing the following after the "Effective date" subheading: "This version of the policy is effective November 1, 2021." and the header "Questions", which should be there.

**Comment by ExperimentsInHonesty** [comment]
@R-Tomas-Gonzalez There needs to be a link to the Figma file where the design can be found

**Comment by JessicaLucindaCheng** [comment]
- Availability: Unsure for the next two weeks (Dec 19 - Jan 2) due to the holidays
- ETA: Tuesday, January 11, 2022

**Comment by JessicaLucindaCheng** [comment]
@IsaacDesigns I'm working on creating the privacy policy page and adding its image to the credits page.
1. For the credits page, is there a specific alt text you want for the privacy policy image?
2. For the privacy policy page, is there a specific alt text you want for the privacy policy image?

Also, can I confirm this [Slack id](https://hackforla.slack.com/team/U0255408P8Q) is yours?

**Comment by JessicaLucindaCheng** [comment]
@R-Tomas-Gonzalez For the privacy-policy.yml, what is the difference between content and type?

**Comment by IsaacDesigns** [comment]
@JessicaLucindaCheng If you are referring to the hero images then you can just null the alt text. 
It's hard to tell with that Slack link but here is my id: U0255408P8Q 
-Isaac Cruz

**Comment by JessicaLucindaCheng** [comment]
@IsaacDesigns Thanks for responding! Is the alt text also null when the image is on the credits page?
<img width="268" alt="privacy-policy-credit" src="https://user-images.githubusercontent.com/31293603/147002887-ab548fe3-427d-424a-9d2f-7b52cee9c68b.png">

**Comment by IsaacDesigns** [comment]
@JessicaLucindaCheng Yes.

**Comment by JessicaLucindaCheng** [comment]
@R-Tomas-Gonzalez Never mind. I found out the difference between content and type. Here is a link: https://github.com/hackforla/website/issues/571#issuecomment-657256359

**Comment by R-Tomas-Gonzalez** [comment]
> @R-Tomas-Gonzalez Never mind. I found out the difference between content and type. Here is a link: [#571 (comment)](https://github.com/hackforla/website/issues/571#issuecomment-657256359)

Great job researching! Thanks for the note! 
Cheers, 
-Tomas

**Comment by JessicaLucindaCheng** [comment]
# Developer's notes

## Label

**Action items**
- [x] Add the url to this the [label description](https://github.com/hackforla/website/labels?q=Privacy+Policy)


## Credits

**Action items**
- [x] Create a privacy-policy directory in assets/images directory.
  - [x] Add privacy-policy.jpg into the privacy-policy directory.
- [x] Create and add privacy-policy.yml to _data/internal/credits directory. This is for the Credits page.
- [x] If https://github.com/hackforla/website/issues/2642 is closed, then update the credits according to the updated template.

**Resources**
- [Credits-yml-file-template](https://github.com/hackforla/website/wiki/Credits-yml-file-template)
- [#571 (comment)](https://github.com/hackforla/website/issues/571#issuecomment-657256359)


## Pre-work
- [x] Go through the text in the Figma desktop version and compare with the Figma mobile version. Comment any differences on the mobile version.
- [x] Go through the text in the Google Doc and compare with the Figma. Comment any differences in the Google Doc.

## Privacy policy page creation
**Action items**
- [x] Add privacy-policy.html to pages directory.
  - [x] Add the content and appropriate html tags
- [x] Add _privacy-policy.scss to _sass/components directory.
    - [x] Apply existing title and paragraph classes to html
    - [x] Apply spacing styling to html for desktop
      - [x] Header
      - [x] Body 
    - [x] Apply spacing styling to html for mobile
      - [x] Header
      - [x] Body 
- [x] Using developer tools, check
  - [x] Check spacing for desktop 1440px wide
  - [x] Check font styling for desktop 1440px wide
  - [x] Check spacing for mobile 375px wide
  - [x] Check font styling for mobile 375px wide
  - [x] Mobile looks good
  - [x] Tablet looks good
  - [x] Desktop looks good    
- [x] Add `@import 'components/privacy-policy.scss';` into the _sass/main.scss file

**List of scss classes used in this page (that are not in _privacy-policy.scss) and where they came from**
- _sass/elements/_containers.scss
  - .header-container
    - .header-text
      - .sub-p
    - .header-hero-image 
  - .flex-container
- _sass/elements/_cards.scss
  - .page-card
  - .card-primary
  - .page-card-lg
- _sass/elements/_typography.scss
  - .title1
  - .title2
  - .paragraph2
- _sass/layouts/_main.scss
  - .content-section

**Notes**

- There are differences in content (words, punctuation, contractions, etc) in the Google Doc vs the Figma desktop design vs the Figma mobile design. These differences are commented on in the Google Doc and Figma. The Figma desktop design was followed when there were differences, except for "Email Addresses" header, and the "Effective date" and "Questions" sections.
  - I used "E-mail Addresses" in the header to be consistent with the beginning of the sentence on the next line, which begins with "E-mail addresses".
  - For the "Effective date" and "Questions" sections, the content in the Google Doc was followed.
- I chose to make all links start with https.

**Resources**

## Tests/Checks

### WCAG
- [x] Use main, section ( and maybe article, nav, header, footer) tags where applicable
- [x] List of links from the Wiki [WCAG 2.0 Compliance Checkers](https://github.com/hackforla/website/wiki/WCAG-2.0-Compliance-Checkers) to use as checkers and tools:
  - [x] [aXe](https://www.deque.com/axe/)  
  - [x] [SiteImprove](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc)  
  - [x] [Tenon](https://tenon.io)  
  - [x] [WAVE](https://wave.webaim.org/extension/)  
  - [x] [Lighthouse](https://developers.google.com/web/tools/lighthouse/#devtools)

### Responsiveness

### Label
- [x] Add the url to this [label description](https://github.com/hackforla/website/labels?q=Privacy+Policy)

**Comment by JessicaLucindaCheng** [comment]
- **Progress:** Placed the hero image in the assets folder and created privacy-policy.yml for credits page
- **Blockers:** None
- **Availability:** Unsure for this week (Dec 26 - Jan 2) due to the holidays
- **ETA:** Tuesday, January 25, 2022 because I forgot to account for the time it takes to test the page on various browsers and for responsiveness in my original ETA

**Comment by JessicaLucindaCheng** [comment]
- Progress: None this week
- Blockers: None
- Availability: 6 hours total by working on it Thu-Fri, Jan 6-7, 2022
- ETA: Tuesday, January 25, 2022

**Comment by JessicaLucindaCheng** [comment]
@R-Tomas-Gonzalez You wrote "Create a corresponding privacy-policy.scss file under /_sass/components directory". Did you mean _privacy-policy.scss?

**Comment by JessicaLucindaCheng** [comment]
- Progress: started on the html page
- Blockers: none
- Availability: 6 hours
- ETA: Tuesday, January 25, 2022

**Comment by R-Tomas-Gonzalez** [comment]
> @R-Tomas-Gonzalez You wrote "Create a corresponding privacy-policy.scss file under /_sass/components directory". Did you mean _privacy-policy.scss?

Yes

**Comment by JessicaLucindaCheng** [comment]
- Progress: Some of the content has been added to the HTML.
- Blockers: This is more of an obstacle/hurdle than a true blocker. I had to restart doing the HTML page as the template I used may not be up to date. I had to pause adding the content to the HTML page when I realized the content in the Figma wasn't exactly the same as the content in the Privacy Policy Draft Google Doc, such as punctuation, contractions, missing header, and a missing section. Also, there were a few discrepancies between the mobile and desktop design in terms of content too. Thus, I had to compare all the content between desktop design, mobile design, and the Privacy Policy draft and make comments as to the differences, which ended up taking most of my time.
- Availability: 6 hours
- ETA: Tuesday, February 1, 2022. I'm adding a week as I didn't account for the obstacle/hurdle I encountered.

**Comment by JessicaLucindaCheng** [comment]
- Progress: None this week
- Blockers: More of another hurdle. Realized that we don't have the paragraph classes defined in the Design System in Figma in our codebase.
- Availability: 6 hours
- ETA: Tuesday, February 8, 2022. I'm adding another week as I didn't account for the classes to not exist, and so I may have to wait for those to be created for me so that I can use them.

**Comment by JessicaLucindaCheng** [comment]
- Progress: Have added all the content (words) to the HTML file. Credit for hero image is added and label is updated with URL. Blank SCSS page was added.
- What still needs to be done: It needs to be styled.
- Blocker/Reason for stopping work on this issue: Need paragraph classes to style the text.

**Comment by JessicaLucindaCheng** [comment]
- Progress: Have added all the content (words) to the HTML file. Credit for hero image is added and label is updated with URL. Blank SCSS page was added.
- Blockers: None
- Availability: 6 hours
- ETA: Tue, March 15, 2022

**Comment by JessicaLucindaCheng** [comment]
Progress: Have added all the content (words) to the HTML file. Credit for hero image is added and label is updated with URL. Blank SCSS page was added.
Blockers: None
Availability: 6 hours
ETA: Tue, March 22, 2022

**Comment by JessicaLucindaCheng** [comment]
Progress: Have added all the content (words) to the HTML file. Credit for hero image is added and label is updated with URL. Blank SCSS page was added.
Blockers: None
Availability: 6 hours
ETA: Tue, April 26, 2022

**Comment by JessicaLucindaCheng** [comment]
Progress: Have added all the content (words) to the HTML file. Credit for hero image is added and label is updated with URL. Blank SCSS page was added. (See [Developer's Notes](https://github.com/hackforla/website/issues/2555#issuecomment-1001147824) for more details.)
Blockers: None
Availability: 6 hours
ETA: Tue, April 26, 2022 but hopefully sooner

**Comment by JessicaLucindaCheng** [comment]
Progress: Have added all the content (words) to the HTML file. Credit for hero image is added and label is updated with URL. Started on the styling of the page: most of desktop is done, need to complete mobile styling. (See [Developer's Notes](https://github.com/hackforla/website/issues/2555#issuecomment-1001147824) for more details.)
Blockers: None
Availability: 6 hours
ETA: Tue, April 26, 2022 but hopefully sooner

**Comment by JessicaLucindaCheng** [comment]
- Progress: Finished with the issue and opened a pr.
- Blockers: None
- Availability: Will update if there are any requests for changes
- ETA: Will update if there are any requests for changes

**Comment by ExperimentsInHonesty** [comment]
Hi Dev Team - Next steps is to make an issue to add the link to the privacy policy page to the bottom of the website.  Currently, the menu says
Credits | Site Map | Join Us
please change it to
Credits | Site Map | Privacy Policy | Join Us
and link Privacy Policy to https://www.hackforla.org/privacy-policy

**Comment by JessicaLucindaCheng** [comment]
@ExperimentsInHonesty 
> Hi Dev Team - Next steps is to make an issue to add the link to the privacy policy page to the bottom of the website. Currently, the menu says Credits | Site Map | Join Us please change it to Credits | Site Map | Privacy Policy | Join Us and link Privacy Policy to https://www.hackforla.org/privacy-policy

- I started issue #3129, which will address your request.

**Comment by JessicaLucindaCheng** [comment]
Sorry, I forgot to put my note here from my pr. I recommend follow up with PM and Design about the notes I left in Figma and Google Docs and also the following:
- There are differences in content (words, punctuation, contractions, etc) in the Google Doc vs the Figma desktop design vs the Figma mobile design. These differences are commented on in the Google Doc and Figma. The Figma desktop design was followed when there were differences, except for "Email Addresses" header, and the "Effective date" and "Questions" sections.
  - I used "E-mail Addresses" in the header to be consistent with the beginning of the sentence on the next line, which begins with "E-mail addresses".
  - For the "Effective date" and "Questions" sections, the content in the Google Doc was followed.

**Comment by JessicaLucindaCheng** [comment]
Reopening this because some things need to be fixed based on meeting with Bonnie.
- [x] Fix the Privacy Policy content to match the content in https://docs.google.com/document/d/19x1KsMTU-QBdopiV08HSBQ0i4g1ChrcPZ8ZXmUK50cY/edit#heading=h.1kvk4qkoxhzw.
- [x] Fix the Google Docs where indicated by suggested edits and comments. Then, resolve the comments.
- [ ] Fix Figma if it is a quick fix. If not a quick fix, pass it to design to fix it. --> Will pass the missing header/section to design
- [x] See if https://github.com/hackforla/website/blob/d98710f82d31a2d9dd6e93590fb36a852689c3f0/_sass/elements/_cards.scss#L66-L68 can be used instead of the variable itself
- [x] See if https://github.com/hackforla/website/blob/gh-pages/_sass/elements/_lists.scss can be used for styling the lists --> Won't be applying those classes because it actually removes the bullet points instead of having default bullet points. So, I will leave the styling for the lists as is.

**Comment by JessicaLucindaCheng** [comment]
- Progress: Still need to fix the issues brought up in https://github.com/hackforla/website/issues/2555#issuecomment-1153288628
- Blockers: None
- Availability: 6 hours each Fri, July 1, 2022 and Sat, July 2, 2022
- ETA: Probably requires 6-12 hours of work. July 9, 2022 (but probably sooner)

**Comment by JessicaLucindaCheng** [comment]
- Progress: Still need to fix the issues brought up in https://github.com/hackforla/website/issues/2555#issuecomment-1153288628
- Blockers: None
- Availability: 6 hours
- ETA: Aug 9, 2022

**Comment by JessicaLucindaCheng** [comment]
I worked on creating the Privacy Policy page and that pull request was merged, so the page is viewable on the website at https://www.hackforla.org/privacy-policy now. However, I had a convseration with Bonnie about discrepancies I noticed in the content in the Google Doc and Figma design. From that conversation with Bonnie, the following changes were requested:
- [x] Edit the Privacy Policy content to match the updated content in this [hackforla.org privacy policy Google Doc](https://docs.google.com/document/d/19x1KsMTU-QBdopiV08HSBQ0i4g1ChrcPZ8ZXmUK50cY/edit#heading=h.1kvk4qkoxhzw)
  - **Note:** Styling should still follow [HFLA Figma - Policy Page](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17845%3A137765). However, the mobile and desktop Figma designs are missing the following after the "Effective date" subheading: "This version of the policy is effective November 1, 2021." and the header "Questions", which should be there.
- [x] Apply this class https://github.com/hackforla/website/blob/d98710f82d31a2d9dd6e93590fb36a852689c3f0/_sass/elements/_cards.scss#L66-L68 and then remove `background: $color-pink` from the _privacy-policy.scss
- [x] Once this pull request #3456 is merged, please do the following:
  - [x] Copy and paste this into a new comment below:
    ```
    Design Team: In the mobile and desktop [Figma design for the Privacy Policy page](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17845%3A137765), the "Effective date" section's content is missing and the "Questions" header is missing. See [this comment in the Google Doc](https://docs.google.com/document/d/19x1KsMTU-QBdopiV08HSBQ0i4g1ChrcPZ8ZXmUK50cY/edit?disco=AAAAbFsfU-4). Please write an issue to fix this.
    
    <details>
      <summary>Click here to see the section that is missing content in the desktop Figma design for the Privacy Policy page</summary>
      <img width="512" alt="" src="https://user-images.githubusercontent.com/31293603/183795111-fb8de903-88ac-4298-b519-c22c04c0f269.png">
    </details>
    
    <details>
      <summary>Click here to see the section that is missing content in the mobile Figma design for the Privacy Policy page</summary>
      <img width="217" alt="" src="https://user-images.githubusercontent.com/31293603/183795115-d8ec9452-59f9-49de-8b75-87c4d1fad8a9.png">
    </details>
    ```
  - [x]  Add the "ready for design lead" label
    - [x] Then, move to the New Issue Approval column
    - [x] Also, let the design lead know either through a message or telling them verbally about it, and giving them a link to the new comment you just wrote. 

### Resources/Instructions
- [HFLA Figma - Policy Page](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17845%3A137765)
- [hackforla.org privacy policy Google Doc](https://docs.google.com/document/d/19x1KsMTU-QBdopiV08HSBQ0i4g1ChrcPZ8ZXmUK50cY/edit#heading=h.1kvk4qkoxhzw)

**Comment by JessicaLucindaCheng** [comment]
- Progress: Completed changes and opened a pr
- Blockers: None
- Availability: Will update if there are changes requested
- ETA: Will update if there are changes requested

**Comment by JessicaLucindaCheng** [comment]
Design Team: In the mobile and desktop [Figma design for the Privacy Policy page](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17845%3A137765), the "Effective date" section's content is missing and the "Questions" header is missing. See [this comment in the Google Doc](https://docs.google.com/document/d/19x1KsMTU-QBdopiV08HSBQ0i4g1ChrcPZ8ZXmUK50cY/edit?disco=AAAAbFsfU-4). Please write an issue to fix this.

<details>
  <summary>Click here to see the section that is missing content in the desktop Figma design for the Privacy Policy page</summary>
  <img width="512" alt="" src="https://user-images.githubusercontent.com/31293603/183795111-fb8de903-88ac-4298-b519-c22c04c0f269.png">
</details>

<details>
  <summary>Click here to see the section that is missing content in the mobile Figma design for the Privacy Policy page</summary>
  <img width="217" alt="" src="https://user-images.githubusercontent.com/31293603/183795115-d8ec9452-59f9-49de-8b75-87c4d1fad8a9.png">
</details>

**Comment by JessicaLucindaCheng** [comment]
> Design Team: In the mobile and desktop [Figma design for the Privacy Policy page](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17845%3A137765), the "Effective date" section's content is missing and the "Questions" header is missing. See [this comment in the Google Doc](https://docs.google.com/document/d/19x1KsMTU-QBdopiV08HSBQ0i4g1ChrcPZ8ZXmUK50cY/edit?disco=AAAAbFsfU-4). Please write an issue to fix this.
> 
> Click here to see the section that is missing content in the desktop Figma design for the Privacy Policy page
> Click here to see the section that is missing content in the mobile Figma design for the Privacy Policy page

- Design issue #3555 has been written to address the discrepancies in Figma.
